src/utils/const/tasks.js

```diff
@@ -1,2 +1,7 @@
 export const CARD_PER_PAGE = 9;
+
+export const TASK_FILTER_ALL = 1;
```

Comment:
Maybe this name need to be changed, otherwise it seems confusing with the name `FILTER_TASK` that I already asked to change to `FILTER_ALL_TASK`